test/extensions/filters/http/header_mutation/header_mutation_integration_test.cc

```diff
@@ -0,0 +1,166 @@
+#include "source/extensions/filters/http/header_mutation/config.h"
+
+#include "test/integration/http_integration.h"
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace HeaderMutation {
+namespace {
+
+class HeaderMutationIntegrationTest : public testing::TestWithParam<Network::Address::IpVersion>,
+                                      public HttpIntegrationTest {
+public:
+  HeaderMutationIntegrationTest()
+      : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, GetParam()) {}
+
+  void initializeFilter() {
+    config_helper_.addFilter(R"EOF(
+name: donwstream-header-mutation
+typed_config:
+  "@type": type.googleapis.com/envoy.extensions.filters.http.header_mutation.v3.HeaderMutation
+  mutations:
+    response_mutations:
```

Comment:
Is there value in also covering request mutations?

Reply:
get it. I will add some request mutations here.